GDEdit Web/GDE.Web/Pages/Index.razor

```diff
@@ -1,5 +1,16 @@
 ﻿@page "/"
+@using GDE.Web.Data
 
 <h1>Hello, world!</h1>
 
 Welcome to your new app.
+
+@code 
+{
+    protected override void OnInit()
+    {
+        base.OnInit();
+        GlobalVariables.CurrentSection = GlobalVariables.LinkItems.Home;
+    }
+
```

Comment:
Empty space

Reply:
thing is Rider wont let me delete the empty space, so blame rider for that one :OMEGALUL:

Reply:
Imagine using Rider

Reply:
Imagine waiting 2 hours for the project to load